4.applications/grit-informed-aggregation/1.generate-profiles/scripts/eval_utils.py

```diff
@@ -0,0 +1,189 @@
+import os
+import glob
+import pandas as pd
+import numpy as np
+import random
+
+from pycytominer.cyto_utils import infer_cp_features
+
+
+def get_featuredata(df):
+    """Return dataframe of output features that we expect to be cell painting features"""
+    return df[infer_cp_features(df, metadata=False)]
+
+
+def get_metadata(df):
+    """Return dataframe of just metadata columns"""
+    return df[infer_cp_features(df, metadata=True)]
+
+
+def percent_score(null_dist, corr_dist, how):
```

Comment:
this is a function borrowed from Niranj, correct? Please add something like:

"Script modified from an original function written by Niranj Chandrasekaran"

Reply:
added in 335d81483e54227b0b9d61373582b124ee1dba2e